BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudFile.cs

```diff
@@ -47,21 +54,27 @@ public BrainCloudFile(BrainCloudClient client)
 #if UNITY_WEBPLAYER || UNITY_WEBGL
             throw new Exception("FileUpload API is not supported on Web builds use FileUploadFromMemory instead");
 #else
-            FileInfo info = new FileInfo(localPath);
-
-            if (!info.Exists)
+            if (Uri.IsWellFormedUriString(localPath, UriKind.Absolute))
             {
                 _client.Log("File at " + localPath + " does not exist");
                 return false;
             }
+            byte[] byteData = GetByteData(localPath);
```

Comment:
that would fail. this function takes filedata as string (wrong)
You need to load the file, and then call upload from memory. It will simplify this function :)
But im not sure I like the idea of keeping the file in memory until you get a response... YOu could load it once response received (Wasn't it like that before?)